mlir/lib/Conversion/TosaToRock/TosaToRock.cpp

```diff
@@ -175,27 +173,14 @@ makeRockConv2D(ConversionPatternRewriter &rw, Operation *op, Value input,
   rock::GemmFeatures features;
   std::tie(arch, num_cu, features) = getArchAttributes(op, input.getType());
 
-  ArrayRef<int64_t> pad64 = pad;
```

Comment:
If we're getting an attribute array, we don't need to make intermediate arrays, to make another attribute array.